src/components/Navbar/Navbar.jsx

```diff
@@ -73,6 +79,9 @@ const roleArray = [
     value : 'developer', label : 'DB - Developer Branch'
   }
 ]
+  const email = "paladbryanj@gmail.com";
```

Comment:
make some algo for this credentials , include also the code sent to email.

Reply:
About the email i've used it's only for the mean time, since i don't have the fetching of email of users from database.